en-US/Language.xml

```diff
@@ -946,6 +946,50 @@ Numbers (move from N entry) -->
 		</para>
 	</section>
 
+<!-- Added section for redundant words -->
+	 <section id="redundant">
+		<title>Avoiding Redundant Words</title>
+		 <para>
+			Avoid redundant words, such as "actually", "just", "really", "simply" and "very". They are filler words that add no value to a sentence.  
```

Comment:
We need to qualify this. Yes, sometimes they are redundant. Sometimes they are not. It depends on how they are used. E.g.,
- "We are proud of our content, and with just cause."
- "Why do you want to do this?" "Just because."

Reply:
Fixed